README.md

```diff
@@ -63,4 +63,9 @@ first need to install both Protocol Buffers and gRPC on your local
 machine. Then you can run `make all` in this directory to generate
 the code.
 
+### Go gRPC Source Code
+It is somewhat difficult to generate Go gRPC source code
```

Comment:
Only nit is - I'm not sure that "somewhat" should be used in this context (it's too colloquial?).